### PR TITLE
Show missions in profile

### LINKED ISSUE #2: [2pt]  Display joined missions in My profile - Filters
Render a list of all joined missions (use `filter()`) on the "My profile" page.

### PR DESCRIPTION
**Pull request by ahangarha**
Now Missions gets shown on the profile page.

I also check if there is any missions already fetched from API. If not, I dispatch fetchMission action to get them.

**:warning: Notice: This PR should be merged after #36 and #35**

closes #2

### CODE REVIEW

File: src/pages/Profile.js [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions } from '../redux/missions/missions';
+import './Profile.css';
 
 function Profile() {
+  const missions = useSelector((state) => (state.missions));
+
+  const dispatch = useDispatch();
+
```
Comment (BonkeGcobo) [inline src/pages/Profile.js:10]: Only missions that should be displayed on the profile should be the joined mission, I believe here you call all the missions,

**Review by BonkeGcobo — CHANGES_REQUESTED** [review]
### Changes Required

Great job soo far, however, there are few changes you need to make before the project is approved

File: src/pages/Profile.js (hunk 1 above)
Comment (ahangarha) [reply src/pages/Profile.js:10]: Good catch :+1:

**Review by BonkeGcobo — APPROVED** [review]
### Approved 🥳 :

Great job in implementing the required Changes it's time to merge it

There is nothing more to say than it is time to merge it
[]

**Pull request merged**